js_modules/dagit/packages/core/src/execute/SolidSelector.tsx

```diff
@@ -159,16 +160,24 @@ export const SolidSelector = (props: ISolidSelectorProps) => {
     }
   };
 
+  if (!data?.pipelineOrError) {
```

Comment:
It does seem like the SolidSelector will "pop" in now instead of just being un-openable while data is loading. Not sure if this is concern or not but this might be a todo to come back and look at when we're polishing

Reply:
Yeah, let's keep an eye on this and see if it's a problem.